x-pack/plugins/uptime/public/state/api/alert_actions.ts

```diff
@@ -119,6 +119,8 @@ function getServiceNowActionParams(): ServiceNowActionParams {
         impact: '2',
         severity: '2',
         urgency: '2',
+        category: null,
```

Comment:
We'll want to update our UI to allow users to actually set these without using the API directly. I'm creating a follow-up issue.

https://github.com/elastic/uptime/issues/300

Reply:
@justinkambic FWIW, there is an API call (subaction) where you can get dynamically all available choices (impact, priority, severity, category, subcategory) for a specific instance. If you are interested to it ping me on Slack!

Example: https://github.com/cnasikas/kibana/blob/itms_category/x-pack/plugins/security_solution/public/cases/components/connectors/servicenow/servicenow_itsm_case_fields.tsx